tests/integration/test.replication.js

```diff
@@ -217,6 +232,9 @@ adapters.forEach(function (adapters) {
         result.docs_written.should.equal(0, 'correct # docs written (1)');
         return db.info();
       }).then(function (info) {
+        if (testUtils.isSyncGateway()) {
+          return [{"ok": {"_id": "doc", "_rev": doc1._rev}}];
+        }
```

Comment:
^ I think this is a case where I would prefer to smooth out the differences in `http.js`. We already do this for Cloudant; they don't actually return `{ok: true}`, whereas I like to be reassured that everything is ok. :)